src/metabase/core.clj

```diff
@@ -44,10 +44,9 @@
 
 ;;; --------------------------------------------------- Lifecycle ----------------------------------------------------
 
-(defn- -init-create-setup-token
-  "Create and set a new setup token and log it."
+(defn print-setup-url
```

Comment:
You should make this private https://github.com/metabase/metabase/wiki/Metabase-Clojure-Style-Guide#make-things-private

The main reason is so if we want to refactor it in the future we don't have to look around the codebase and make sure it's not used anywhere else. (Also when you look thru this namespace to use something from it the surface area of it is a little smaller and it's clearer which things are mean to be used elsewhere)

Reply:
Good catch, thanks!